src/elisp/treemacs-icons.el

```diff
@@ -302,7 +302,7 @@ Necessary since root icons are not rectangular."
     (treemacs-create-icon :file "json.png"          :extensions ("json"))
     (treemacs-create-icon :file "julia.png"         :extensions ("jl"))
     (treemacs-create-icon :file "elx.png"           :extensions ("ex"))
-    (treemacs-create-icon :file "elx-light.png"     :extensions ("exs" "eex"))
+    (treemacs-create-icon :file "elx-light.png"     :extensions ("exs" "eex" "leex"))
```

Comment:
Please prefix the commit message with "[Icons]".

Reply:
@Alexander-Miller Sorry I forgot, it has been modified👌🏻️